learnyounode/06-http-client.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+let http = require('http');
+
+http.get(process.argv[2], function(res) {
+  res.setEncoding('utf8');
+  res.on('data', function(chunk) {
+    console.log(chunk);
+  });
```

Comment:
Mmm.. That's not quite right. You'll want to collect all of the chunks of data and concatenate them before printing them out.